hammers/osrest/keystone.py

```diff
@@ -62,7 +62,7 @@ def users(auth, enabled=None, name=None):
     if enabled is not None:
         params['enabled'] = enabled
 
-    response = API.get(auth, '/users', params=params)
+    response = API.get(auth, '/v3/users', params=params)
```

Comment:
There are a lot of other API.get references without the v3 suffix. Does it make sense to just put that /v3 on the `auth` param? Which, I guess, is in the config, or?

I imagine for example that the project lookup functions are borked.

Reply:
Also, how was this working before? :D

Reply:
This wasn't working before. No other hammers were using osrest for keystone users.